{"hierarchy":{"paths":[["doc:\/\/SwiftPy\/documentation\/SwiftPy","doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftpy\/pythonerror\/zerodivisionerror(_:)"]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"ZeroDivisionError"},{"kind":"text","text":"("},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":")"}],"languages":["swift"]}]}],"metadata":{"externalID":"s:7SwiftPy11PythonErrorO012ZeroDivisionD0yACSScACmF","symbolKind":"case","roleHeading":"Case","title":"PythonError.ZeroDivisionError(_:)","modules":[{"name":"SwiftPy"}],"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ZeroDivisionError","kind":"identifier"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":")","kind":"text"}]},"kind":"symbol","identifier":{"url":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/ZeroDivisionError(_:)","interfaceLanguage":"swift"},"references":{"doc://SwiftPy/documentation/SwiftPy":{"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy","title":"SwiftPy","role":"collection","url":"\/documentation\/swiftpy","abstract":[],"kind":"symbol","type":"topic"},"doc://SwiftPy/documentation/SwiftPy/PythonError/ZeroDivisionError(_:)":{"identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError\/ZeroDivisionError(_:)","role":"symbol","type":"topic","title":"PythonError.ZeroDivisionError(_:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ZeroDivisionError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":")","kind":"text"}],"url":"\/documentation\/swiftpy\/pythonerror\/zerodivisionerror(_:)","abstract":[],"kind":"symbol"},"doc://SwiftPy/documentation/SwiftPy/PythonError":{"abstract":[],"role":"symbol","type":"topic","navigatorTitle":[{"text":"PythonError","kind":"identifier"}],"kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"PythonError","kind":"identifier"}],"title":"PythonError","url":"\/documentation\/swiftpy\/pythonerror","identifier":"doc:\/\/SwiftPy\/documentation\/SwiftPy\/PythonError"}}}